.eslintrc.json

```diff
@@ -32,6 +34,7 @@
     "no-invalid-this": "off",
     "no-unused-expressions": ["error", { "allowShortCircuit": true }],
     "react/jsx-key": "off",
-    "react/display-name": "off"
+    "react/display-name": "off",
+    "jsx-a11y/label-has-for": "off"
```

Comment:
Since we use the `FieldContainer` in many areas it's a pain to disable this everywhere. Don't think it will cause too much of an issue.